api/views.py

```diff
@@ -37,3 +38,25 @@ class QuestionCategoryViewSet(viewsets.ReadOnlyModelViewSet):
 class UserStatsViewSet(viewsets.ReadOnlyModelViewSet):
     queryset = MyUser.objects.all()
     serializer_class = UserStatsSerializer
+
+
+class ActionsViewSet(viewsets.ReadOnlyModelViewSet):
```

Comment:
you need to set up permissions for this view set. Unauthenticated users should receive 401.